koku/reporting/provider/azure/models.py

```diff
@@ -50,7 +49,6 @@
     "partnumber": "",
     "paygprice": 0.0,
     "planname": "",
-    "pretaxcost": 0.0,
```

Comment:
There is a few field names in here that can be removed. `usagedatetime` and `currency`.